components/contact/Card.vue

```diff
@@ -0,0 +1,41 @@
+<script setup lang="ts">
+  defineProps<{
+    content: {
+      name: string
+      image: string
+      position: string
+      email: string
+      phone: string
+    }
+  }>()
+</script>
+
+<template>
+  <div class="d-flex flex-column">
+    <VCard class="mx-auto" width="435" variant="flat">
```

Comment:
Should remove auto margin. Margins should be added when the component is used instead.